### PR TITLE
Integrate plus projections and minus projections into current chart

### PR DESCRIPTION
**Pull request by dgendill**
Migrated the BaseChart to typescript so we can separate out chart code from the boilerplate.  Added projection plus and projection minus data to chart.  Moved existing styles into main.scss. Added js-compiled folder for compiled js asets.  Removed old files that accidentally made it into previous merges.  Removed compiled assets from the repo.

### CODE REVIEW

File: code/projections.html [hunk 1]
```diff
@@ -43,18 +81,14 @@
 
   </script>
   <link rel="stylesheet" type="text/css" href="css/bootstrap/bootstrap.css"/>
```
Comment (abettermap) [inline code/projections.html]: @dgendill I think this line needs to go.

**Comment by abettermap** [comment]
*Stats.js* isn't building on `npm start` but 616af32 fixes it. Evidently `npm build` is not the same as `npm run build` for me.

**Comment by abettermap** [comment]
I think just drop that Bootstrap line and I'll merge the PR. I didn't look at your `js` changes super hard but I didn't get any errors besides the problems I mentioned above. Also I didn't know `scss` could import `css` like you did with Bootstrap. Neato!

**Comment by dgendill** [comment]
Bootstrap line removed.

**Comment by abettermap** [comment]
Sorry, your PR got lost in the Slack/GH flood of notifications. 🌊

**Comment by dgendill** [comment]
No problem :) thanks

**Pull request merged**